data/json/items/armor/coats.json

```diff
@@ -921,10 +1331,7 @@
       { "covers": [ "leg_l", "leg_r" ], "coverage": 90, "encumbrance": [ 2, 2 ] },
       { "covers": [ "arm_l", "arm_r" ], "coverage": 90, "encumbrance": [ 2, 2 ] }
     ],
-    "pocket_data": [
-      { "pocket_type": "CONTAINER", "max_contains_volume": "500 ml", "max_contains_weight": "3 kg", "moves": 80 },
-      { "pocket_type": "CONTAINER", "max_contains_volume": "500 ml", "max_contains_weight": "3 kg", "moves": 80 }
-    ],
+    "pocket_data": [ { "pocket_type": "CONTAINER", "max_contains_volume": "500 ml", "max_contains_weight": "1 kg", "moves": 80 } ],
```

Comment:
Deleted a pocket here? No max length?

Reply:
Kimono was given a small pocket to represent being able to tuck some small items into the sash.  I've seen enough knife wielding kimono wearers in the movies to know they're hiding that thing somewhere.